src/constants/Permissions.js

```diff
@@ -77,4 +77,5 @@ export const Permissions = {
   WaistCircumference: 'WaistCircumference',
   Weight: 'Weight',
   Workout: 'Workout',
+  Summary: "Summary"
```

Comment:
What do you think of calling this "Activity Summary"?

Reply:
Sounds good, fixed